module/VuFind/tests/unit-tests/src/VuFindTest/Search/Solr/ResultsTest.php

```diff
@@ -76,7 +76,10 @@ public function testCursorMark(): void
      */
     public function testFacetTranslations(): void
     {
-        $mockTranslator = $this->createMock(TranslatorInterface::class);
+        $mockTranslator
+            = $this->getMockBuilder(TranslatorInterface::class)
+            ->addMethods(['getLocale'])
+            ->getMockForAbstractClass();
```

Comment:
...and another opportunity to use the trait. This test is specifying `exactly(2)` instead of `any` calls to the translate method, but we could address that by adding another parameter to the `getMockTranslator` method -- i.e.

```
protected function getMockTranslator($translations, $expectation = null)
{
...
    $translator->expects($expectation ?? $this->any())->method('translate')
```

Reply:
Using the trait here as well. I didn't see why using "exactly(2)" instead of "any" is important for this test. So, I ignored this for now. But, if you think this is required I will change it.

Reply:
It's a slightly more specific assertion, but I agree that it's probbbly not a hugely important distinction here.